daisy/step.go

```diff
@@ -250,7 +250,14 @@ func (s *Step) populate(ctx context.Context) dErr {
 	return err
 }
 
+func (s *Step) recordStepTime(startTime time.Time) {
+	endTime := time.Now()
```

Comment:
Can be inlined

Reply:
I want to keep it this way to make it easier to read. Otherwise, a pair of (startTime, time.Now()) doesn't have a obvious meaning.